src/main/java/services/WriterServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package services;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+import model.Product;
+
+public class WriterServiceImpl implements WriterService {
+    private static final String INITIAL_FORMAT = "name,quantity\n";
```

Comment:
```suggestion
    private static final String HEADER = "name,quantity\n";
```
don't use \n, use System.lineSeparator()